src/components/ADempiere/Field/OptionsInField/operatorComparison/index.vue

```diff
@@ -0,0 +1,134 @@
+<!--
```

Comment:
Please rename this to **FieldOptions**

Reply:
Rename from OptionsInFiend to FieldOptions